File: Invest-parent/Invest-Invest/Invest-Invest-Repository/src/main/java/com/illunex/invest/invest/entity/ExamItemTemplate.java
package com.illunex.invest.invest.entity;

import com.illunex.invest.invest.entity.enumable.ViewMode;

import javax.persistence.*;

@Entity
public class ExamItemTemplate {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long examItemTemplateIdx;
    String title;
    ViewMode viewMode;

    @ManyToOne
    ExamMenu examMenu;
}

File: microservices/invest/src/main/kotlin/com/illunex/invest/invest/persistence/entity/ExamItemTemplate.kt
package com.illunex.invest.invest.persistence.entity

import com.illunex.invest.invest.persistence.entity.enumable.ViewMode
import javax.persistence.*

@Entity
public class ExamItemTemplate {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var examItemTemplateIdx: Long? = null
    var title: String? = null
    var viewMode: ViewMode? = null

    @ManyToOne
    var examMenu: ExamMenu? = null
}
